src/sentry/static/sentry/app/components/avatar/baseAvatar.jsx

```diff
@@ -164,6 +165,8 @@ export default BaseAvatar;
 // sensible default.
 const StyledBaseAvatar = styled('span')`
   flex-shrink: 0;
+  background-color: ${p => p.theme.whiteDark};
```

Comment:
This is so that there is a "placeholder" that's light grayish while we load an image avatar.